pandas/core/nanops.py

```diff
@@ -812,15 +812,21 @@ def unique1d(values):
     return uniques
 
 
-def _checked_add_with_arr(arr, b):
+def _checked_add_with_arr(arr, b, arr_nans=None, b_nans=None):
```

Comment:
Sure but you do understand that that isn't what we're getting when doing TDI addition / subtraction.  In addition, there is no need to check for `isnull` since the work already done for us during instantiation of the TDI instance.